app/lib/features/auth/providers/notifiers/auth_notifier.dart

```diff
@@ -17,7 +16,6 @@ class AuthStateNotifier extends StateNotifier<bool> {
 
   Future<void> nuke(BuildContext context) async {
     await ActerSdk.nuke();
-    ref.invalidate(spacesProvider);
```

Comment:
You are correct.
Actually I thought forced update of spaces provider would be meaningless because `nuke()` is called when user was logged out and he is going to delete old caches.